config/glide.php

```diff
@@ -41,4 +41,5 @@
         'dpr' => '1',
     ],
     'presets' => [],
+    'original_media_for_extensions' => [],
```

Comment:
With the addition of this key, `add_params_to_svg` is not in use anymore, which could be producing a breaking change. Can we consider it in the implementation so that the behavior defaults to include the SVG extension in the `original_media_for_extensions`? When set to true, the extension should not be included.